homeassistant/components/ezviz/strings.json

```diff
@@ -92,6 +92,17 @@
           }
         }
       }
+    },
+    "service_depreciation_sound_alarm": {
+      "title": "Ezviz Sound alarm service is being removed",
+      "fix_flow": {
+        "step": {
+          "confirm": {
+            "title": "[%key:component::ezviz::issues::service_depreciation_sound_alarm::title%]",
+            "description": "Ezviz Sound alarm service is deprecated and will be removed.\nTo sound the alarm, you can instead use the `siren.toggle` service targetting the Siren entity.\n\nPlease remove the use of this service from your automations and scripts and select **submit** to close this issue."
```

Comment:
```suggestion
            "description": "Ezviz Sound alarm service is deprecated and will be removed in 2024.3.\nTo sound the alarm, you can instead use the `siren.toggle` service targeting the Siren entity.\n\nPlease remove the use of this service from your automations and scripts and select **submit** to fix this issue."
```

Reply:
Should the date be in the description as well? Was previously requested to remove it (in ther other recent pulls) so just making sure.

Reply:
No, my mistake as it's already mentioned by itself. Remove that part

Reply:
No problem. One last thing, I see targetting is UK spelling and targeting is US spelling. Is there a specific standard that home assistant follows? Should I submit a pull to fix the rest to US spelling or do we leave it on UK as the rest in the stings file?

Apologies for wasting your time with this question.

Reply:
Base is US English so would be good to correct it.